src/workspace/Lesson/Section/Editor.jsx

```diff
@@ -43,80 +45,45 @@ export const EditorWrapper = styled.div`
 	}
 `
 
-export const editorTheme = {
-	editor: {
-		primary: {
-			background: theme.colors.primary,
-		},
-	},
-	plugins: {
-		rows: {
-			menu: {
-				highlightColor: theme.colors.primary,
-				dropzone: {
-					highlightColor: theme.colors.primary,
-				},
-			},
-		},
-		text: {
-			hoverColor: "#B6B6B6",
-		},
-	},
+const getInitialDocValue = (section) => {
+	const { docValue } = section;
+	// if error try catch
+	return (docValue && Object.keys(docValue).length) ? docValue : { plugin: "rows" }
 }
 
+const canWrite = (section, props) => {
+	// if section not exist or has no key, setError
+	return section.scopePermission === 'write' && props.editing
+}
 
 export const Editor = (props) => {
```

Comment:
must not part of this pr but mabey if we start to refector we could write here the vars itself with {}? So everybody can see witch props are used or could be set